docs/getting-started/self-hosting.md

```diff
@@ -121,6 +125,53 @@ docker-compose up -d
  docker-compose logs dashboard
 ```
 
+### Advanced: Running netbird behind an existing reverse-proxy
+
+If you want to run netbird behind your own reverse-proxy, some additional configuration-steps have to be taken to [Step 2](#step-2--prepare-configuration-files).
+
+:::info
+Not all reverse-proxies are supported as netbird uses *gRPC* for various components.
+:::
+
+#### Configuration for netbird
+
+In `setup.env`:
+- Set ```NETBIRD_DOMAIN``` to your domain, e.g.  `demo.netbird.io`
+- Set ```NETBIRD_DISABLE_LETSENCRYPT=true```
+
+In `base.env`:
```

Comment:
Let's avoid updating the base.env as it updates on every git pull. I've created a small PR that will help with that: https://github.com/netbirdio/netbird/pull/783